04.ls/ls_command.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+COLUMN_COUNT = 3
+
+def main
+  files = fetch_files
+  puts format_to_show(files)
+end
+
+def fetch_files
+  Dir.foreach(dir_path).filter_map { |file| file unless file.start_with?('.') }
+end
+
+def dir_path
+  ARGV[0] || '.'
+end
+
+def format_to_show(files)
+  sorted_files = files.sort
+
+  row_count = (files.size / COLUMN_COUNT.to_f).ceil
+
+  column_width = sorted_files.max_by(&:length).length
+
+  formatted_strings = []
+  row_count.times do |row|
+    same_row_files = []
+    COLUMN_COUNT.times do |col|
```

Comment:
個人の好みですが、こういう命名の方が読みやすいような気がしました。いかがでしょう？

```suggestion
  rows = []
  row_count.times do |row_idx|
    columns = []
    COLUMN_COUNT.times do |col_idx|
```

Reply:
そうですね。
コードの動きとしては、"各列のファイル名を行ごとにまとめる"なので提案していただいた`rows`と`columns`の方が分かりやすいと思います！